traffic_control/models/road_marking.py

```diff
@@ -116,6 +116,13 @@ class AbstractRoadMarking(SourceControlModel, SoftDeleteModel, UserControlModel,
         blank=True,
         null=True,
     )
+    device_type = models.ForeignKey(
+        TrafficControlDeviceType,
+        verbose_name=_("Device type"),
+        on_delete=models.PROTECT,
+        limit_choices_to=Q(Q(target_model=None) | Q(target_model=DeviceTypeTargetModel.ROAD_MARKING)),
+        help_text=_("Type of the device from Helsinki Design Manual."),
```

Comment:
Seems like you must run `makemigrations`

Reply:
Only difference is a capital letter in verbose_name, and migrations are created :/